implementations/rust/ockam/ockam_command/src/tcp/inlet/show.rs

```diff
@@ -17,7 +17,7 @@ use crate::{fmt_ok, Result};
 const PREVIEW_TAG: &str = include_str!("../../static/preview_tag.txt");
 const AFTER_LONG_HELP: &str = include_str!("./static/show/after_long_help.txt");
 
-/// Delete a TCP Inlet
+/// Show a TCP inlet's details
```

Comment:
To keep the consistency with the other commands, let's keep the `Inlet` word capitalized.